notifications/09-20-23-1500.md

```diff
@@ -0,0 +1,67 @@
+**What is happening?**
+
+CHG0052476 - MCS SILVER - 2023 Q4 Patching - Hard Prune of Image registry, ETCD Defrag and Trident Upgrade
+
+CHG0052425 - MCS SILVER - 2023 Q4 Patching - Upgrade to OCP 4.13.12
+
+CHG0052477 - MCS GOLD - 2023 Q4 Patching - Hard Prune of Image registry, ETCD Defrag and Trident Upgrade
+
+CHG0052479 - MCS GOLD - 2023 Q4 Patching - Upgrade to OCP 4.13.12
+
+CHG0052478 - MCS GOLDDR - 2023 Q4 Patching - Hard Prune of Image registry, ETCD Defrag and Trident Upgrade
+
+CHG0052480 - MCS GOLDDR - 2023 Q4 Patching - Upgrade to OCP 4.13.12
+
+These six changes represent the work involved to upgrade the production clusters from Openshift 4.12 to 4.13.
+
+**When?**
+
+CHG0052476 - MCS SILVER - 2023 Q4 Patching - Hard Prune of Image registry, ETCD Defrag and Trident Upgrade will commence at 0600 on Sunday October 15th and run as late as 0900.
+
+CHG0052425 - MCS SILVER - 2023 Q4 Patching - Upgrade to OCP 4.13.12 will commence at 9:00am on Monday October 16th. Upgrade efforts will be allowed to proceed between 9:00am and 5:00pm, and be paused after-hours to ensure developers are present during node drains to look after their apps if needed.
```

Comment:
I recommend adding when the change window ends for each RFC involving regular Openshift upgrades. It's more or less done for the Sunday morning RFCs.

Reply:
I'll add the end dates in a moment.